CHANGES.md

```diff
@@ -1,3 +1,25 @@
+# 4.1.0 (2022-11-22)
```

Comment:
because we add a new `feature` to the method, I prefer updating the minor version instead of the patch version